internal/framework/service/workers_for_platforms/resource.go

```diff
@@ -0,0 +1,134 @@
+
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	namespace, err := r.client.CreateWorkersForPlatformsDispatchNamespace(ctx, cloudflare.AccountIdentifier(data.AccountID.ValueString()),
+		cloudflare.CreateWorkersForPlatformsDispatchNamespaceParams{
+			Name: data.Name.ValueString(),
+		},
+	)
+	if err != nil {
+		resp.Diagnostics.AddError("failed to create Workers for Platforms namespace", err.Error())
+		return
+	}
+	data.ID = types.StringValue(namespace.Result.NamespaceName)
+	data.Name = types.StringValue(namespace.Result.NamespaceName)
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func (r *WorkersForPlatformsResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	var data *WorkersForPlatformsNamespaceModel
+
+	resp.Diagnostics.Append(req.State.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	namespace, err := r.client.GetWorkersForPlatformsDispatchNamespace(ctx, cloudflare.AccountIdentifier(data.AccountID.ValueString()), data.ID.ValueString())
+	if err != nil {
+		resp.Diagnostics.AddError("failed reading Workers for Platforms namespace", err.Error())
+		return
+	}
+	data.ID = types.StringValue(namespace.Result.NamespaceName)
+	data.Name = types.StringValue(namespace.Result.NamespaceName)
+	resp.Diagnostics.Append(resp.State.Set(ctx, &data)...)
+}
+
+func (r *WorkersForPlatformsResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	var data *WorkersForPlatformsNamespaceModel
+
+	resp.Diagnostics.Append(req.Plan.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	resp.Diagnostics.AddError("failed to update Workers for Platforms namespace", "Not implemented")
+}
+
+func (r *WorkersForPlatformsResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	var data *WorkersForPlatformsNamespaceModel
+
+	resp.Diagnostics.Append(req.State.Get(ctx, &data)...)
+
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	err := r.client.DeleteWorkersForPlatformsDispatchNamespace(ctx, cloudflare.AccountIdentifier(data.AccountID.ValueString()), data.Name.ValueString())
+
+	if err != nil {
+		resp.Diagnostics.AddError("failed to delete Workers for Platforms namespace", err.Error())
+		return
+	}
+}
+
+func (r *WorkersForPlatformsResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
+	idparts := strings.Split(req.ID, "/")
+	if len(idparts) != 2 {
+		resp.Diagnostics.AddError("error importing Workers for Platforms namespace", "invalid ID specified. Please specify the ID as \"account_id/name\"")
+		return
+	}
+	resp.Diagnostics.Append(resp.State.SetAttribute(
+		ctx, path.Root("account_id"), idparts[0],
+	)...)
+	// TODO: Figure out if I can set name here, ID is a UUID and isn't one we really display/use user-facing wise
```

Comment:
is this still a TODO? you can set anything you want as the ID however, it shouldn't be based on anything the user can change (i.e. the name). if you must use a use configurable, you'll need to make sure we force replace it whenever it changes like you already have defined in the schema.

Reply:
Oops nah forgot to remove it. It's using the name now.